js/libs/keycloak-admin-client/README.md

```diff
@@ -123,6 +123,27 @@ Then start the tests with:
 pnpm test
 ```
 
+## methods
+
+### registerTokenProvider
+
+The term "keycloakAdminClient" refers to an instance of the Keycloak Admin Client library. Keycloak is an open-source identity and access management solution, and the Admin Client is a component that provides programmatic access to the administrative functionalities of a Keycloak server.
```

Comment:
I think this is already pretty self explanatory from the earlier examples in the README, no need to repeat things.
```suggestion
```